src/open_producten/producten/serializers/product.py

```diff
@@ -18,14 +22,56 @@ class ProductSerializer(serializers.ModelSerializer):
     product_type_id = serializers.PrimaryKeyRelatedField(
         write_only=True, queryset=ProductType.objects.all(), source="product_type"
     )
+    eigenaren = EigenaarSerializer(many=True)
 
     class Meta:
         model = Product
         fields = "__all__"
         validators = [
-            BsnOrKvkValidator(),
             DateValidator(),
             StatusValidator(),
             VerbruiksObjectValidator(),
             DataObjectValidator(),
+            NestedObjectsValidator("eigenaren", Eigenaar),
         ]
+
+    def validate_eigenaren(self, eigenaren: list[Eigenaar]) -> list[Eigenaar]:
+        if len(eigenaren) == 0:
+            raise serializers.ValidationError(_("Er is minimaal één eigenaar vereist."))
+        return eigenaren
+
+    @transaction.atomic()
+    def create(self, validated_data):
+        eigenaren = validated_data.pop("eigenaren", [])
+
+        product = super().create(validated_data)
+
+        for eigenaar in eigenaren:
+            EigenaarSerializer().create(eigenaar | {"product": product})
```

Comment:
What happens if you pass a uuid for an Eigenaar in a POST request?

Reply:
Thx, forgot about that, because of id = serializers.UUIDField(required=False) it will be passed through and saved. id is now removed if it exists.